cpp/dolfinx/io/ADIOS2_utils.cpp

```diff
@@ -0,0 +1,36 @@
+// Copyright (C) 2024 Abdullah Mujahid, Jørgen S. Dokken and Garth N. Wells
```

Comment:
Please remove Garth unless I missed something

Reply:
This file is adapted (almost verbatim) from the ADIOS2Writers.*
Hence the Copyright header was extended.